monorepo/packages/core/README.md

```diff
@@ -0,0 +1,34 @@
+# Pangolin Core
+Building components to use in your react app, such Box, Dropdown, Modal, and etc.
+
+## Installation
+`yarn add @pangolindex/core`
+
+or
+
+`npm install @pangolindex/core`
+
+### Install below dependancies as its peer dependancies
+
+```
+react
+react-dom
+@pangolindex/sdk
```

Comment:
@SarjuHansaliya @prodesert22 does it make sense to also move this to Honeycomb Github org?

For example, how would we store Ortege chain details? We can't store them in the Pangolin SDK

Reply:
Yes we will move to honeycomb, are a task about this
https://github.com/pangolindex/components/pull/404

Reply:
@HariSeldon23 ortege sdk will be under `@ortege/sdk` and chain details we will store in ortege sdk.